.github/workflows/sample.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Function to display usage
+usage() {
+    echo "Usage: $0 [ --b | -base <base_branch>] [ --h | -head <head_branch>]"
```

Comment:
Shall we match the parameter names used in the CI template (e.g. 'base' and 'starting'), or we could also change the starting branch name to match these parameters.